app/build.gradle

```diff
@@ -53,9 +53,11 @@ dependencies {
     implementation lib.androidxlifecycle
     implementation lib.recyclerview
     implementation lib.cardview
+    implementation lib.stetho
     implementation lib.koinandroidxviewmodel
     implementation lib.koinandroid
     implementation lib.koinandroidxscope
+    implementation lib.stetho
```

Comment:
Are you sure that you need this line two times?

Reply:
I missed it, thanks Mau, removed